Sources/WelcomeToGitHub/QuickMaths.swift

```diff
@@ -25,7 +25,7 @@ struct QuickMaths {
 	///   - rhs: Right hand side of the equation
 	/// - Returns: subtraction of lhs and rhs as an Int.
 	func subtract(_ lhs:Int, _ rhs:Int ) -> Int {
-		return 0
+		return lhs - rhs
```

Comment:
Someone else is working on this feature! 
```suggestion
		return 0
```